### DISCUSSION TITLE
I have some problem with the install

### DISCUSSION THREAD

**Discussion by Zerbaib** [TomSchimansky/CustomTkinter#1960] [General]
I'm not i a venv:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/81523999/cc79b3f3-43e3-4ddb-a4ec-095bd70b4ce0)

And if i'm in a venv:
![image](https://github.com/TomSchimansky/CustomTkinter/assets/81523999/b12fcc54-637d-42a8-b4c8-f5d84d75ec8f)

I have tryed with `sudo apt install python3-customtkinter` and not work
BUT if i am on vscode in the project repo i can launch the project

**Comment by Akascape** [comment]
Another duplicate: https://github.com/TomSchimansky/CustomTkinter/issues/1915
Search there
